setup/templates/index.html

```diff
@@ -0,0 +1,166 @@
+<!doctype html>
```

Comment:
Not really, template should be stored somewhere where binary is running. Since setup folder contains supporting files which are copied to same directory I can't really think of a better place. 
It makes sense to leverage //go:embed functionality from latest go 1.16 but this means it will be required to build the stuff - this way template will be embeded onto the binary. Is it ok for you?

Reply:
Ah I see, my assumption is that nobody that isn't also working on the codebase would be running this so it should be safe to put it in there. But this is fair and seems like a good use case for this embedding feature, give it a try

Reply:
Done - please try it out again. Looks rather cool IMHO. Template is embedded and no template copy necessary in folder e.g. binary is self-sufficient. Moved template out of setup folder too.